posts/2024-11-28_seoul-retreat/README.md

```diff
@@ -0,0 +1,67 @@
+---
+publication_date: 2024-11-28T00:00:00Z
+slug: seoul-retreat
+tags: [gnoland, ecosystem, meetup, retreat, seoul]
+authors: [michelleellen]
+---
+
+# Gnomes spotted in Seoul: Onbloc x Core Engineering Collaboration 
+
+gno.land’s core engineering team has just wrapped up an intense and highly 
+productive two-week coding sprint with our core contributor, 
+[Onbloc](https://onbloc.xyz/). Hosted in Onbloc's offices in the heart of Seoul,
+this onsite collaboration stands as one of our most successful development cycles
+yet, setting new records for pull requests merged and advancing critical priorities
+for our upcoming mainnet launch.
+
+[![office](https://gnolang.github.io/blog/2024-11-28_seoul-retreat/src/thumbs/office.jpg)](https://gnolang.github.io/blog/2024-11-28_seoul-retreat/src/office.jpg)
+
+## Starting with a Bang: test5 and engineering developments
+
+The collaboration kicked off with the launch of test5. This milestone set the
+tone for two weeks of unparalleled productivity. During this time, the teams 
+focused on benchmarking and gas cost estimation, aiming to optimize transaction
+efficiency for both users and developers.
+
+A significant priority was advancing `gnoweb`, gno.land’s universal web frontend. 
+The team outlined a development [roadmap](https://github.com/gnolang/gno/issues/3191) 
+for iterative feature releases, ensuring adaptability and a user-centric experience 
+throughout the process. Key features include:
+- Design Overhaul: Creating a more user-friendly UI with a color palette 
+matching gno.land's true spirit.
+- UX Redesign: Establishing `gnoweb` as the go-to realm web frontend.
+- Code Refactoring: Enhancing security and maintainability.
+- Navigation Improvements: Simplifying user navigation and introducing a query 
+system for direct linking to specific `gnoweb` content or commands.
+
+Finally, the team collaborated closely with Onbloc’s DevOps experts to streamline 
```

Comment:
We can add more things here, `gnoweb` was just one project:

https://hackmd.io/@JFGdLdZISka23j6-v8UYbA/H17Awcf7Jg/edit


We also helped Onbloc identify pain points in current Gnoswap, Adena and Gnoscan implementations, through a workshop with both teams